### PR TITLE
Fixing content nitpicks

### PR DESCRIPTION
**Pull request by mbelton-buildkite**
A general pass to align the README content with the current template.

### CODE REVIEW

File: bazel-ci/README.md [hunk 1]
```diff
@@ -11,6 +11,31 @@ primary_emojis: [":bazel:"]
...
+After you select **Use template**, you’ll:
+
+1. Connect the Git repository with your Bazel project.
+2. Set the build and test parameters for your Bazel project.
+3. Configure the compute—run locally, on-premises, or in the cloud.
+4. Run the pipeline.
+
+You can then play around with the pipeline settings. For example, run the pipeline locally while you iterate on the definition or set a schedule to trigger a nightly build.
+
+For more information about advanced Bazel usage, see [How Bazel built its CI system on top of Buildkite](https://buildkite.com/blog/how-bazel-built-its-ci-system-on-top-of-buildkite).
```
Comment (dannymidnight) [inline bazel-ci/README.md:39]: ❤️

File: golang-cross-compile/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Compile Go binary"
-description: "Build Go binaries against different operating systems, architectures and versions of Go."
+description: "Build Go binaries against different operating systems, architectures, and versions of Go."
```
Comment (dannymidnight) [inline golang-cross-compile/README.md:3]: Reading this description now, I reckon we could be more specific. e.g. 

```suggestion
description: "Build and cross-compile a Go binary for Linux, Windows, and Darwin architectures.
```

**Review by dannymidnight — APPROVED** [review]
Lovin' it! Thanks @mbelton-buildkite

**Review by olyism — APPROVED** [review]
Thanks @mbelton-buildkite this is much clearer and I love positive/actionable use of language.

File: golang-cross-compile/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Compile Go binary"
-description: "Build Go binaries against different operating systems, architectures and versions of Go."
+description: "Build and cross-compile a Go binary for Linux, Windows, and Darwin architectures.
```
Comment (mbelton-buildkite) [inline golang-cross-compile/README.md:3]: Oops, missing closing quote now:

```suggestion
description: "Build and cross-compile a Go binary for Linux, Windows, and Darwin architectures."
```

**Pull request merged**